src/containers/Root/AuthorizedRoot.tsx

```diff
@@ -17,23 +18,31 @@ const ContentContainer = styled.div`
   display: flex;
 `;
 
-export const AuthorizedRoot: React.FC = () => {
+interface IAuthorizedRootProps {
+  onLogoutClick: () => void;
+  onLanguageClick: () => void;
+  onAddUserClick: () => void;
+}
+
+export const AuthorizedRoot: React.FC<IAuthorizedRootProps> = props => {
+  const { onLogoutClick, onLanguageClick, onAddUserClick } = props;
+
   return (
     <Wrapper>
       <Header
-        onLogoutClick={ () => {} }
-        onChangeLanguage={ () => {} }
+        onLogoutClick={ onLogoutClick }
+        onChangeLanguage={ onLanguageClick }
         userName='Roman Tsiuapiak'
       />
       <Wrapper>
         <ContentContainer>
           <Switch>
-            <Route path='/' component={ UsersContainer }/>
+            <Route path='/:id?' component={ UsersContainer }/>
```

Comment:
we use this container for displaying all users? why have you decided to add id here?